cpp/src/arrow/flight/server.h

```diff
@@ -25,6 +25,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <grpcpp/support/time.h>
```

Comment:
We can't expose gRPC symbols publically.

Reply:
The Shutdown method is templated: https://grpc.github.io/grpc/cpp/classgrpc_1_1_server_interface.html#a644772d11101318f1424e595dee73ccf and converts things via [TimePoint](https://grpc.github.io/grpc/cpp/classgrpc_1_1_time_point.html) which states it'll take a std::chrono::system_clock::time_point.

Reply:
Yeah, that makes way more sense. Changed, please review.